frontend/src/metabase/dashboard/components/DashboardHeader/DashboardHeader.tsx

```diff
@@ -612,7 +612,7 @@ export const DashboardHeader = (props: DashboardHeaderProps) => {
           key="expand"
           aria-label={t`Enter Fullscreen`}
           icon="expand"
-          className={cx("text-brand-hover", CS.cursorPointer)}
+          className={CS.cursorPointer}
```

Comment:
Why can this class be removed instead of migrated?

Reply:
The class never existed